### ISSUE TITLE
Addition of `postcss-opacity-percentage` breaks on node < 16

### ISSUE THREAD

**Issue by prarabdhb** [csstools/postcss-plugins#218]
### Bug description

With the addition of `postcss-opacity-percentage` in v7.3.0, the package cannot be installed on node versions < 16. This should either be released as a major upgrade, or should've been mentioned in the changelog as a breaking change. 

### Plugin version

7.3.0

### What OS are you experiencing this on?

macOS

### Node Version

14.18.1

### Validations

- [X] Follow our [Code of Conduct](https://github.com/csstools/postcss-plugins/blob/master/CODE_OF_CONDUCT.md)
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by romainmenke** [comment]
@prarabdhb Can you add a bit more info?
Error output or something?

We have tests running in node 12 and 14 and these passed : https://github.com/csstools/postcss-plugins/actions/runs/1771534168

**Comment by Mitsunee** [comment]
Also having this issue in my [fgo-tools](https://github.com/mitsunee/fgo-tools/) repo when trying to upgrade postcss-preset-env:

```shell
$ yarn add -D postcss-preset-env@latest

yarn add v1.22.17
[1/4] Resolving packages...
[2/4] Fetching packages...
error postcss-opacity-percentage@1.1.1: The engine "node" is incompatible with this module. 
Expected version ">=16". Got "14.18.2"
error Found incompatible module.
```

I currently need to stay on either node12 or node14 as vercel does not yet support node16.

**Comment by prarabdhb** [comment]
> @prarabdhb Can you add a bit more info? Error output or something?
> 
> We have tests running in node 12 and 14 and these passed : https://github.com/csstools/postcss-plugins/actions/runs/1771534168

Hey, sorry to miss out on the logs. 
Thanks to @Mitsunee , we also see the same output.

This is most likely because `postcss-opacity-percentage` expects node >= 16.

**Comment by pzrq** [comment]
I'll add that this is also affecting [CRA](https://github.com/facebook/create-react-app) / react-scripts downstream:

```
$ yarn add -D react-scripts@5.0.0
yarn add v1.22.17
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error postcss-opacity-percentage@1.1.1: The engine "node" is incompatible with this module. Expected version ">=16". Got "14.18.2"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by Antonio-Laguna** [comment]
@pzrq for the time being, could you try with?

```json
"resolutions": {
  "postcss-preset-env": "7.2.3"
}
```

**Comment by Antonio-Laguna** [comment]
The new version of opacity percentage has been released, ideally getting rid of the lock, the resolves, and node_modules should bring the new dependency.

Could you verify? 

@pzrq @prarabdhb @Mitsunee 

Will upgrade and release 7.3.1 right after that

**Comment by Mitsunee** [comment]
Everything fine here on my end now, reran `yarn add -D postcss-preset-env@latest` and the upgrade finished without any errors

**Comment by prarabdhb** [comment]
Yes, everything works on our end as well. Thanks for the quick resolution!

**Comment by pzrq** [comment]
> Could you verify?
> 
> @pzrq ...

@Antonio-Laguna Thank you for the fast release, I can confirm it's working for me now on node 14 👍🏼

**Comment by Antonio-Laguna** [comment]
This is finally released as [7.3.1](https://github.com/csstools/postcss-plugins/blob/main/plugin-packs/postcss-preset-env/CHANGELOG.md#731-february-3-2022) !

Thanks all for your patience and all of the information you shared here! ❤️
